packages/client/src/api/patches.ts

```diff
@@ -90,13 +91,14 @@ export const patchAPI = API => {
     return await enrichRows(rows, tableId)
   }
 
-  // Wipe any HBS formulae from table definitions, as these interfere with
+  // Wipe any HBS formulas from table definitions, as these interfere with
```

Comment:
NAB: this is also correct English - both are correct plurals of formula. Just in case you didn't know!

Reply:
TIL that. I had no idea, thx :)